packages/renderer/src/lib/ui/TerminalWindow.svelte

```diff
@@ -28,7 +31,14 @@ async function refreshTerminal(): Promise<void> {
     TerminalSettings.SectionName + '.' + TerminalSettings.LineHeight,
   );
 
-  terminal = new Terminal({ fontSize, lineHeight, disableStdin: true, theme: getTerminalTheme() });
+  terminal = new Terminal({
+    fontSize,
+    lineHeight,
+    disableStdin: disableStdIn,
```

Comment:
This changes `disableStdin` from `true` to `undefined` for current instances of `TerminalWindow`s. Is it expected?

Reply:
@feloy Ohhh, you are right. There were a few places in my other PR that didn't have disableStdin at all. I'll change it back to default true and add in the other PR false for this parameter

Reply:
@feloy fixed!